### ISSUE TITLE
Research on best file size, type, quality, ... integrate results in manual

### ISSUE THREAD

**Issue by wimallegaert** [ices-eg/SmartDots#298]
Set-up a test event for performance and quality testing of the images. List of samples linked to the same images in a different resolution, file format, quality, ....
Compare file size, response time, ...
Use results of the tests for the guidelines on the pictures in the web application manual

**Comment by wimallegaert** [comment]
![image](https://user-images.githubusercontent.com/13795097/204999320-edc5aad8-c22e-4f83-b0ea-f543a59d5309.png)

**Comment by JCoadDavies** [comment]
Feedback example

[SmartDots image size test_Feedback_Julie.xlsx](https://github.com/ices-eg/SmartDots/files/10140982/SmartDots.image.size.test_Feedback_Julie.xlsx)

**Comment by ComeDenechaud** [comment]
Here are my comments for the test set with some side notes: 
[SmartDots.image.size.test_Feedback_Julie_Côme.xlsx](https://github.com/ices-eg/SmartDots/files/10164140/SmartDots.image.size.test_Feedback_Julie_Come.xlsx)

**Comment by kbekaert** [comment]
[SmartDots.image.size.test_Feedback_Julie_Come_Karen (1).xlsx](https://github.com/ices-eg/SmartDots/files/10165028/SmartDots.image.size.test_Feedback_Julie_Come_Karen.1.xlsx)

**Comment by gudrunfinnb** [comment]
[SmartDots.image.size.test_Feedback_Julie_Come_Karen_Gudrun.xlsx](https://github.com/ices-eg/SmartDots/files/10165465/SmartDots.image.size.test_Feedback_Julie_Come_Karen_Gudrun.xlsx)

**Comment by MKAlbisua** [comment]
And here mines.
[SmartDots.image.size.test_Feedback_Juliex2_Come_Karen_Gudrun_Maria.xlsx](https://github.com/ices-eg/SmartDots/files/10173995/SmartDots.image.size.test_Feedback_Juliex2_Come_Karen_Gudrun_Maria.xlsx)

**Comment by marcinblaszkowski** [comment]
[SmartDots.image.size.test_Feedback_Juliex2_Come_Karen_Gudrun_Maria_Marcin.xlsx](https://github.com/ices-eg/SmartDots/files/10179725/SmartDots.image.size.test_Feedback_Juliex2_Come_Karen_Gudrun_Maria_Marcin.xlsx)

Hi All,
My comments included.
Kind regards
Marcin

**Comment by JCoadDavies** [comment]
The summary that Wim wrote about tells us what we need to know. And it has been very useful to see this test. Thanks for the input :-)
What I find hard is how to write some concrete guidelines around this for those who do not know or understand the related terminology. We cannot scare people away from taking images for SmartDots 
I think that Wim's suggestion to automatically adjust file type, file size and resolution during upload might be the most reliable solution.
We can still give guidelines, e.g.:
1. The object in the image should always take up the largest area and empty space around the object should be avoided as much as possible 
2. The image has to be in .jpg format
3. Resolution should be below 100 
4. File size should be below 1500 KB
5. Large images should be compressed (and give a table with examples here...)

All input welcome 👍

**Comment by rellebo** [comment]
Yes it is a good solution to make adjustments to the images during the upload.

**Just for the point 1** for the sclerochronology (to be checked for the other parts).
I think it is necessary to keep the same image field for a given species. So for a young fish there will be a lot of empty space on the image and for an old fish the otolith is bigger and so we minimize the empty space.

For performance issues, we could consider the use of an _adaptive threshold_ algorithm detecting the otolith... in the image and standardize the set of empty space pixels to 0 or 255 (in 8bit). To see the amount of development work for the performance gain. 
To facilitate the work of the algo, it is necessary to give an advice of acquisition on the most uniform background possible white in transmission or black in reflection. 
@++

**Comment by JCoadDavies** [comment]
**I agree on point 1**
The problem with inexperienced age readers not having a grasp on the development of otolith size with age is only confounded by working with images as opposed to working with physical samples. We need to avoid this by increasing the magnification used when photographing the smaller otoliths. In the past we have recommended that all samples within a set of otolith images are taken at the same magnification to avoid this exact issue (and also avoid incorrect calibration when using different magnifications). 
I like the suggestion of an adaptive threshold algorithm (in the future) but this has its challenges when working with otoliths with a translucent zone at the edge. We can add in the guidelines that the background should be as uniform in colour as possible.

Anymore comments on this? 
Can someone make a table with some examples taken from the excel? 
And should we give a guideline on the image dimensions?

**Comment by rellebo** [comment]
At Ifremer we have this type of table saved in the parameters of the image acquisition software .
![ImageSizeSP](https://user-images.githubusercontent.com/119053715/212010193-ac68115b-cec2-419a-8b57-c47ca4ef77d4.jpg)

**Comment by UlrikaBeier** [comment]
I agree with Julie that suggestions here should be made into clear guidelines, such as these (copied from above):
 1. The object in the image should always take up the largest area and empty space around the object should be avoided as much as possible
 2. The image has to be in .jpg format
 3. Resolution should be below 100
 4. File size should be below 1500 KB
 5. Large images should be compressed (and give a table with examples here...)

I checked quickly with the present 
WGSMART_Draft Guidelines for image quality in SmartDots events.docx
sent out 25 August by Julie for comments (due 6 October). 
My question is if the input from here is already implemented in the guidelines document?
